srtcore/core.cpp

```diff
@@ -10589,6 +10683,8 @@ int srt::CUDT::processData(CUnit* in_unit)
         HLOGC(qrlog.Debug, log << CONID() << "WILL REPORT LOSSES (filter): " << Printable(filter_loss_seqs));
         sendLossReport(filter_loss_seqs);
 
+        // XXX unsure as to whether this should change anything in the TSBPD conditions.
+        // Might be that this trigger is not necessary.
```

Comment:
This signal is indeed no longer necessary because the TSBPD thread now gets notified when an earlier packet has been received.